modules/AWS/eks/main.tf

```diff
@@ -11,6 +11,9 @@ module "eks" {
   vpc_id  = var.vpc_id
   subnets = var.subnets
 
+  kubeconfig_aws_authenticator_command = "aws"
```

Comment:
Can you please add the comment regarding the removal of these variables in eks module v 18? Ideally, link https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/UPGRADE-18.0.md